includes/functions.php

```diff
@@ -150,13 +162,17 @@ function fields_states() {
 /**
  * Get supported layouts for web stories.
  *
+ * @since 1.5.0
+ * @access private
+ * @todo Move out of this file
+ *
  * @return mixed|void
  */
 function get_layouts() {
```

Comment:
As noted before, this could live in theme supports trait. What use is this to a developer?

Reply:
We can do this in another PR, even after release.